src/it/MWAR-450/verify.bsh

```diff
@@ -0,0 +1,94 @@
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.*;
+
+import org.codehaus.plexus.util.*;
+
+boolean result = true;
+
+try
+{
+    File target = new File( basedir, "target/mwar450/WEB-INF/classes" );
+    if ( !target.exists() || !target.isDirectory() )
+    {
+        System.err.println( "target/mwar450/WEB-INF/classes is missing or is not a directory." );
+        return false;
+    }
+
+    // Load and check log4j.xml
+    File log4jxml = new File( target, "log4j.xml" );
+    if ( !log4jxml.exists() || log4jxml.isDirectory() )
+    {
+        System.err.println( "log4j.xml is missing or is a directory." );
+        return false;
+    }
+    FileInputStream fis = new FileInputStream ( log4jxml );
+    String paramContent = IOUtil.toString ( fis, "UTF-8" );
+    System.out.println( "content='" + paramContent + "'" );
```

Comment:
This output will be mangled if run on non-UTF-8.

Reply:
We have tried this on both Windows (CP-1252) and Linux (UTF-8), and the output from the integration test looks fine for us, without mangled characters. Which platform did you have problems on?